plugins/vuex-persist.js

```diff
@@ -1,5 +1,9 @@
 import VuexPersistence from 'vuex-persist';
 
 export default ({ store }) => {
-    new VuexPersistence({}).plugin(store);
+    new VuexPersistence({
+        filter: (mutation) => {
```

Comment:
@jgnasser I tried a few different approaches for ensuring the assessment search text gets cleared until I landed on opting us out of persisting the assessment search state to browser storage. 

Ideally the app wouldnt store all/most of its state in browser storage (it should only store what is explicitly needed), however now that we have wired it up to store almost all state by default, its hard to undo without risking regression bugs. 

As we encounter more similar bugs with persisted state, the solution may be to similarly opt pieces of state out of being copied to browser storage